nat-lab/pyproject.toml

```diff
@@ -0,0 +1,35 @@
+[project]
+name = "nat-lab"
+
+[tool.black]
+preview = true
+skip_magic_trailing_comma = true
+exclude="tests/protobuf"
+
+[tool.isort]
+profile = "black"
+force_single_line = false
+no_inline_sort = true
+sort_reexports = true
+float_to_top = true
+combine_star = true
+remove_redundant_aliases = true
+combine_as_imports = true
+no_sections = true
+skip=["tests/protobuf"]
+
+[tool.autoflake]
+remove-unused-variables = true
+remove-all-unused-imports = true
+remove-duplicate-keys = true
+ignore-init-module-imports = true
+recursive = true
+in-place = true
+
+[tool.pylint]
+max-line-length = 88
+ignore-paths="tests/protobuf/*"
+
+[tool.pylint.'MESSAGES CONTROL']
+recursive = true
+disable = "C0103,C0114,C0115,C0116,E0115,W1510,W0719,C0301,R0915,R0914,C0411,R0913,R0902,R0916,W0511,R0904,R0903,E1101,W0603,W0702,R0912,R0801"
```

Comment:
Could you please add a comment above this line describing those? Sth like:
```
# C0103 - Used when the name doesn't conform to naming rules associated to its type (constant, variable, class...).
# C0114 - Used when a module has no docstring. Empty modules do not require a docstring.
# E0115 - Emitted when a name is both nonlocal and global.
...
```

Reply:
replaced with the name of message instead of code, should be more clear... don't wanna clutter file with full lines of comments

Reply:
It's fine. Thanks. Maybe you could also create JIRA issue to try to shorten this list as the next improvement step

Reply:
Sure thing -> LLT-4091